catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/OrderRequestImpl.java

```diff
@@ -171,26 +175,39 @@ public State complete(DeliveryManifestHolder deliveryManifestHolder)
                 if (order.del_list != null) {
                     for (DeliveryDetails deliveryDetails : order.del_list) {
                         Destination destination = deliveryDetails.dests;
-                        FileLocation httpDestination = destination.f_dest();
-                        if (httpDestination != null) {
-                            String outputName = filename;
-
-                            if (StringUtils.isBlank(outputName)) {
-                                if (StringUtils.isNotBlank(httpDestination.file_name)) {
-                                    outputName = httpDestination.file_name;
-                                } else {
-                                    outputName = UUID.randomUUID()
-                                            .toString();
+
+                        try {
+                            FileLocation httpDestination = destination.f_dest();
+
+                            if (httpDestination != null) {
+                                String outputName = filename;
+
+                                if (StringUtils.isBlank(outputName)) {
+                                    if (StringUtils.isNotBlank(httpDestination.file_name)) {
+                                        outputName = httpDestination.file_name;
+                                    } else {
+                                        outputName = UUID.randomUUID()
+                                                .toString();
+                                    }
                                 }
+
+                                List<String> filesSent = writeData(httpDestination,
+                                        packageFormatType,
+                                        files,
+                                        outputName);
+                                PackageElement packageElement = new PackageElement();
+                                packageElement.files =
+                                        filesSent.toArray(new String[filesSent.size()]);
+                                packageElements.add(packageElement);
                             }
+                        } catch (BAD_OPERATION e) {
+                            LOGGER.error("Bad CORBA operation", e);
```

Comment:
I think it should be debug. CorbaExceptionUtils doesn't do anything special for BAD_OPERATION, as that seems to be a standard exception class, so it's not as necessary here.